docs/feature/type-features.md

```diff
@@ -4,22 +4,23 @@ The following document describes Type entity features.
 
 ## Additional references to parent nodes
 
-> **NOTE** This feature is currently not implemented.
-> In the future, this feature will be also available for Attribute.
+:::info
+In the future, this feature will be also available for Attribute.
+:::
 
 Type can contain additional references to other parent nodes. The connection means that the Type becomes a child of the referenced parent nodes. In a result, the Type has multiple parents.
 
 ![Additional references to parent nodes](./assets/type-additional-refs.svg)
 
+Parent nodes can be any path under `cap.type.` or `cap.core.type` as long as it doesn't point to the concrete Type, such as `cap.core.type.platform.kubernetes`.
+
 Currently, the feature brings the following benefits:
 - **GraphQL API:** it allows User to find related Types based on a prefix of the parent node.
 - **Implementation manifest:** the **requires** property in Implementation manifest can refer to Types, which has additional references to parent node
 
 ### Find Types based on prefix of parent nodes
 
-> **NOTE** This feature is currently not implemented.
-
-1. `cap.type.platform.cloud-foundry` references `cap.core.type.platform` empty node:
+1. `cap.type.platform.cloud-foundry` references `cap.core.type.platform` parent node:
```

Comment:
If you used Nomad in the diagram, maybe it's good to also replace all Cloud Foundry occurrences in the documentation to Nomad, for the sake of consistency. What do you think?

I know we will still keep cloud foundry manifest inside hub manifest, unless you want to update that place as well. It would be better to have Nomad in hub manifests as well, but not sure if it is really worth it...

Reply:
changed to `nomad` in all places, `hub-manifests` will be updated here: https://github.com/capactio/hub-manifests/pull/55/files (this PR is not directly connected with it, but the change is so small that IMO there is no need for separate PR)